event.coffee

```diff
@@ -88,6 +88,7 @@ class Event extends Model
     "<!date^#{@starts_at_stamp}^{date_pretty} at {time}|#{@starts_at}>"
 
   @getter 'slack_where', ->
+    return "<#{@location}>" if URL_REGEX.test(@location)
```

Comment:
might be nice to add comments:
```coffee
  @getter 'slack_where', ->
    # does it look like a URL?
    return "<#{@location}>" if URL_REGEX.test @location

    # probably an address, let's search for it
    "<#{@map_url}|#{@venue}>" if @map_url
```

might also be fun to try to handle addresses / plain text differently
```coffee
  @getter 'slack_where', ->
    # does it look like a URL?
    return "<#{@location}>" if URL_REGEX.test @location

    # does it look like an address? (is there a number?)
    return "<#{@map_url}|#{@venue}>" if /\d/.test @location

    # I'm out of ideas, just print the text
    @location
```
(would need a test to see if this works)

Reply:
That's a good point. Added a fallback & `/\d/` checking :+1: